deployments/overlays/ingress/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+resources:
+- ../../base
+# Uncomment to add custom css as a configmap
+# - ../../css 
+- ingress.yaml
+
+metadata:
+  annotations:
+    name: jwt-inspect
+  labels:
+    app: jwt-inspect
+namespace: open-microtools
```

Comment:
I think these section is not necessary to have it here. it mean to have overlay for ingress for this folder